de.tubs.variantsync.core/src/de/tubs/variantsync/core/view/featurecontext/FeatureContextHandler.java

```diff
@@ -17,10 +17,12 @@
 public class FeatureContextHandler implements IHandler {
 
 	@Override
-	public void addHandlerListener(IHandlerListener handlerListener) {}
+	public void addHandlerListener(IHandlerListener handlerListener) {
+	}
```

Comment:
I guess this is stubbing for futur work, please document in dedicated issue.

Reply:
This problem will be addressed by the following issue https://github.com/tthuem/VariantSync/issues/57